helm-deployment/ldap-subchart/deployment.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+cd certs
+./genereate_ldap_certs.sh
+
+kubectl create secret generic ldap-certs --from-file=ldap.crt --from-file=ldap.key --from-file=ca.crt
```

Comment:
To be consistent with other pull request reviews, our certificates ecosystem and way of installing platform this secret should be creating using helm.